src/components/AudioPlayer/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Text from 'antd/lib/typography/Text'
+import { forwardRef } from 'react'
+import PlayPauseIcon from './components/PlayPauseIcon'
+import useWaveSurfer from './helpers/useWaveSurfer'
+import PlaybackSpeed from './components/PlaybackSpeed'
+import { Button } from '../Button/index'
+import { Icon } from '../Icon/index'
+import { getDisplayTime } from './helpers/utils'
+import './styles.scss'
+
+const AudioPlayer = forwardRef((props, ref) => {
+  const { className, url } = props
+  const { waveSurfer, playerConfig, durationConfig } = useWaveSurfer(url)
+
+  const { isPlaying, loading } = playerConfig
+  const { totalDuration, currentDuration } = durationConfig
+
+  return (
```

Comment:
implement error handling for this component so that no new error causes app crash

Reply:
@RitikSrivastava-Plivo added todo for this